lib/kitchen/provisioner/chef_base.rb

```diff
@@ -108,7 +108,12 @@ def init_command
       end
 
       def install_command
-        "sudo /bin/sh #{File.join(config[:root_path], 'installer.sh')}"
+        # TODO: We should check use_sudo? in here
+        if powershell_shell?
+          "Powershell.exe #{installer_script_path}"
```

Comment:
Can we add a `TODO` here to check this with JayM and see if there is a better way of doing it.